lib/index.js

```diff
@@ -111,16 +114,17 @@ function mergeOptions(options) {
   options = options || {};
   const mergedOpts = assign({}, defaults, options);
   const easyImportOpts = mergedOpts['postcss-easy-import'];
-  const origTransform = easyImportOpts.transform;
```

Comment:
should we add a warning here so that we can avoid a major release?

```js
if (easyImportOpts.transform && !easyImportOpts.load) {
  console.warn('Suitcss uses `postcss-easy-import` v2 which renames `transform` to `load`')
}
```